From 806225a7d2fa0290a02f0ada1adedf99519099b4 Mon Sep 17 00:00:00 2001
From: Yuwei Chen <yuwei.chen@intel.com>
Date: Mon, 3 Aug 2020 11:00:32 +0800
Subject: [PATCH] Test case2 for Change build option under BuildOptions.IA32

Description: Change build option under BuildOptions.IA32 (Add "-D CRB_FLAG") in TestPkg/buildoptions.dsc.

Repo: edk2
Signed-off-by: Yuwei Chen <yuwei.chen@intel.com>
---
 TestPkg/buildoptions.dsc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/TestPkg/buildoptions.dsc b/TestPkg/buildoptions.dsc
index 9ca9e101ff..10f601abf6 100644
--- a/TestPkg/buildoptions.dsc
+++ b/TestPkg/buildoptions.dsc
@@ -33,7 +33,7 @@
   CLANGPDB:*_*_*_DLINK_FLAGS = /ALIGN:4096
   
 [BuildOptions.IA32]
-  RELEASE_MYTOOLS_IA32_CC_FLAGS        = /D MDEPKG_NDEBUG
+  RELEASE_MYTOOLS_IA32_CC_FLAGS        = /D MDEPKG_NDEBUG -D CRB_FLAG
 
 [BuildOptions.IA32.EDKII.PEIM]
   GCC:*_*_*_DLINK_FLAGS = -z common-page-size=0x1000
-- 
2.27.0.windows.1

